ontology/yaml/resources/GATEWAYS/entity_types/GW.yaml

```diff
@@ -1209,4 +1209,4 @@ LINX:
   - supply_water_temperature_setpoint_5
   - outside_air_relative_humidity_sensor
   - heating_supply_water_temperature_setpoint
-  
```

Comment:
revert? otherwise it will keep bouncing?

Reply:
we reverted on the cl, this is going to happen regardless :(